src/platform/unittest/DataPathTest.cpp

```diff
@@ -506,11 +508,13 @@ struct CxPlatSocket {
         _In_ uint16_t PartitionId = 0
         ) const noexcept
     {
+        CXPLAT_ROUTE Route;
+        Route.LocalAddress = LocalAddress;
+        Route.RemoteAddress = RemoteAddress;
```

Comment:
While it would be a larger change, I think it's more future proof to update this `Send` function to take the `Route`; otherwise you're just going to have to change it later when we start adding more to the route.

Reply:
Discussed offline; will leave this as-is for now.